monkey/monkey_island/cc/services/config_schema.py

```diff
@@ -381,6 +399,7 @@
                             "title": "Harvest Azure Credentials",
                             "type": "boolean",
                             "default": True,
+                            "attack_techniques": ["T1110", "T1078"],
```

Comment:
why is T1110 in here? we can use brute force without harvesting credentials

Reply:
Sorry, I forgot to revert after tests. What I think would be best is to add all of the techniques at once so we wouldn't have to go back each time and think about what config value uses which technique . Current code supports schema with not yet implemented techniques.

Reply:
let's talk about it on slack